src/main/scala/com/yotpo/metorikku/output/writers/file/FileOutputWriter.scala

```diff
@@ -2,10 +2,13 @@ package com.yotpo.metorikku.output.writers.file
 
 import com.yotpo.metorikku.configuration.job.Streaming
 import com.yotpo.metorikku.configuration.job.output.File
+import com.yotpo.metorikku.exceptions.MetorikkuWriteFailedException
 import com.yotpo.metorikku.output.Writer
 import org.apache.log4j.LogManager
 import org.apache.spark.sql.{DataFrame, DataFrameWriter, SparkSession}
 
+// scalastyle:off cyclomatic.complexity
+// scalastyle:off method.length
```

Comment:
This practice tends to be contagious :)
Consider removing the scalastyle:off
If it results in too big a refactor, and it is too late in the game, maybe merge and we can fix it later

Reply:
You're right,
I added it because after my change the ```write``` function was too long